server/index.js

```diff
@@ -102,6 +102,18 @@ app.put('/qa/answers/:answerId/report', (req, res) => {
     });
 });
 
+app.get('/qa/questions/:id', (req, res) => {
+  const { id } = req.params;
+
+  atelier.getQuestions(id)
+    .then((data) => {
+      res.status(201).send(data);
```

Comment:
Status code 200 (OK) is probably better for this

Reply:
I'll change it to 200